tools/ota_e2e_tests/README.md

```diff
@@ -2,43 +2,80 @@
+    * Set the **Default region name** as the same region you will create OTA update jobs from, example: `us-west-2`
     * Set the **Default output format** as `json`
 
 ## Board Configuration Setup
 
 The test scripts need information unique to your board and your AWS account. This is contained in a **board.json**.
 
-1. See **board.json** in the same directory as this README for information on each field. Create one or more for each board you wish to run tests on.   
+1. See **board.json** in the same directory as this README for information on each field. Create one or more for each board you wish to run tests on.
 Some items are pre-filled, please verify the configuration is sufficient to your needs.
 
 2. Some tests are designed to check that un-trusted certificates do not work. If you do not want to generate your own throw-away certificate/key pairs,
-example certificates are provided, for signing schemes using RSA SHA256 and ECDSA SHA256, under the 'credentials' directory. The names of the files 
-indicate the signing algorithm. Import the certificates with the algorithm applicable to your device into AWS Certificate Manager. Use the resulting ARN for 
+example certificates are provided, for signing schemes using RSA SHA256 and ECDSA SHA256, under the 'credentials' directory. The names of the files
+indicate the signing algorithm. Import the certificates with the algorithm applicable to your device into AWS Certificate Manager. Use the resulting ARN for
 the `"aws_untrusted_signer_certificate_arn"` board config value.
 
 ## Running the tests
 
-* Run all of the tests on board specified in board.json:  
+* Run all of the tests on board specified in board.json:
 `python ota_e2e.py --board-configs board.json`
 
-* Run all of the tests on directory of board configurations:  
+* Run all of the tests on directory of board configurations:
 `python ota_e2e.py --board-config-dir C:/example_dir/board_configs/`
-  
-* Run only the specified tests:  
-`python ota_e2e.py --board-configs board.json --tests OtaTestGreaterVersion`  
+
+* Run only the specified tests:
+`python ota_e2e.py --board-configs board.json --tests OtaTestGreaterVersion`
 The names of the available tests are pre-filled in board.json under the `"supported_tests"` field.
-  
-* More options specified in --help:  
+
+* More options specified in --help:
 `python ota_e2e.py --help`
+
+## FAQ
+
+### How to create a self-signed code signing certificate
+
+1. Make sure [openssl](https://www.openssl.org/) is installed and can be accessed from commandline.
+1. Use the following text to create a file named `cert_config.txt` in your workspace. Replace
+   `test_signer@amazon.com` with your email address.
+
+    ```txt
+    [ req ]
+    prompt             = no
+    distinguished_name = my dn
```

Comment:
You might use something like:

`distinguished_name = <your distinguished name here>`

...so that the reader knows not to just put this in verbatim.